source/class/qx/theme/tangible/ColorEngine.js

```diff
@@ -87,7 +87,7 @@ var helper = {
     var color = d[0];
     var state = d[1];
     switch (state) {
-      case "focused":
+      case "focussed":
```

Comment:
The original was the correct spelling

Reply:
hm, apparently it's both - ill put it back

Reply:
oh hang on - i think this was someone else's change, to bring the name into line with the rest of Qooxdoo (it's already in the CHANGELOG and nothing else uses double "ss" except a few tests

Reply:
Oh, I guess that's what your last comment was saying, too.

Reply:
that's where this must have started from, the "focused" state not being detected.  Iv'e just realised I only compared the source/class directory so that's why the CHANGELOG is not being reflected